release-notes/aws/10.0.0-rc1

```diff
@@ -0,0 +1,29 @@
+## :zap:  Giant Swarm Release 10.0.0-rc1 for AWS is now active for you! :zap:
+
+We are pleased to announce that Node Pools is now released! 
+
+Please see here the supported material which we recomend reading before getting started: https://docs.giantswarm.io/basics/nodepools
+
+We also kindly remind you that from this release and for a period of SIX months onwards, to give you a chance to move your clusters, we will be supporting both the 'Legacy' and the 'New' branches separately which will include:
```

Comment:
Imo you need to first explain that people will not be able to migrate <10.0.0 clusters to the node pools release. This is why we support the 9.0.0 branch for another 6 months.

Reply:
done, please review changes